xbmc/guilib/GUITextLayout.cpp

```diff
@@ -540,75 +542,93 @@ void CGUITextLayout::WrapText(const vecText &text, float maxWidth)
   if (!m_font)
     return;
 
-  int nMaxLines = (m_maxHeight > 0 && m_font->GetLineHeight() > 0)?(int)ceilf(m_maxHeight / m_font->GetLineHeight()):-1;
-
   m_lines.clear();
 
+  size_t nMaxLines;
+  if (m_maxHeight > 0 && m_font->GetLineHeight() > 0)
+  {
+    nMaxLines = static_cast<size_t>(std::ceil(m_maxHeight / m_font->GetLineHeight()));
+    if (nMaxLines == 0)
```

Comment:
Can this actually happen? since we' re in a block with m_maxHeight >0, ceil should return values > 1?

Reply:
its true removed